### PR TITLE
config: runtime: tests: Add v4l2 fluster tests lava template

### PR DESCRIPTION
**Pull request by musamaanjum**
Copy file template file v4l2-decoder-conformance.jinja2 and edit it.

Note: This is my first PR. Please review it and I'll improve it.

### CODE REVIEW

**Comment by musamaanjum** [comment]
I've added the staging-skip until someone reviews to not disturb the staging.

**Comment by laura-nao** [comment]
Looks ok to me as it's basically just a copy of the old template (except for `plan` being replaced with `node.name`). I'll let someone from the core team review and approve this too.

File: config/runtime/tests/v4l2-decoder-conformance.jinja2 [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{%- if testsuite -%}
+    {% set testsuite_arg = "-ts " + testsuite %}
+{%- endif -%}
+{%- if decoders -%}
+    {%- set decoder_list = [ ] -%}
+    {%- for decoder in decoders -%}
+        {{ decoder_list.append(decoder) or "" }}
+    {%- endfor -%}
+    {%- set decoders_arg = "-d " + decoder_list|join(" ") -%}
```
Comment (a-wai) [inline config/runtime/tests/v4l2-decoder-conformance.jinja2:9]:
```suggestion
    {%- set decoders_arg = "-d " + decoders|join(" ") -%}
```

`decoders` is already a list.

File: config/runtime/tests/v4l2-decoder-conformance.jinja2 [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          os:
+          - debian
+          scope:
+          - functional
+        run:
+          steps:
+          - python3 /usr/bin/fluster_parser.py {{ testsuite_arg }} {{ decoders_arg }} {{ videodec_timeout_arg }} {{ videodec_parallel_jobs_arg }}
+      from: inline
+      name: {{ plan }}
+      path: inline/{{ plan }}.yaml
```
Comment (a-wai) [inline config/runtime/tests/v4l2-decoder-conformance.jinja2:35]: Where is `plan` defined? I suspect this was available in the legacy system but not something we have in the new system.

**Review by a-wai — CHANGES_REQUESTED** [review]

File: config/runtime/tests/v4l2-decoder-conformance.jinja2 (hunk 2 above)
Comment (musamaanjum) [reply config/runtime/tests/v4l2-decoder-conformance.jinja2:31]: @laura-nao @nuclearcat any ideas about this?

Comment (laura-nao) [reply config/runtime/tests/v4l2-decoder-conformance.jinja2:31]: Right, I missed this. `plan` doesn't seem to be used by the new system indeed, I think we can use `node.name`  instead here as well.

Comment (musamaanjum) [reply config/runtime/tests/v4l2-decoder-conformance.jinja2:31]: Okay. Put name: node.name and removed path.

Comment (laura-nao) [reply config/runtime/tests/v4l2-decoder-conformance.jinja2]: @musamaanjum the path is a requirement for inline test definitions in LAVA (https://docs.lavasoftware.org/lava/writing-tests.html#using-inline-test-definitions), I think we should keep it and use `inline/{{ node.name }}.yaml`.

Comment (musamaanjum) [reply config/runtime/tests/v4l2-decoder-conformance.jinja2]: Updated the path.

**Comment by nuclearcat** [comment]
Thank you @musamaanjum , this is very demanded test, tested on staging, merging.

**Pull request merged**